.github/workflows/ci.yml

```diff
@@ -36,9 +36,18 @@ jobs:
           TOXENV: ${{ matrix.toxenv }}
         run: tox
 
+      - name: Run Integration Tests 
+        run: |
+            cd ..
+            git clone https://github.com/openedx/devstack
```

Comment:
I was getting an error in the CI on make `dev.clone.https command`: `make: *** No rule to make target 'dev.clone.https'.  Stop.`
I also reproduced the error locally. The command works fine in the `openedx/devstack` repository.
 I replaced `https://github.com/edx/devstack` with `https://github.com/openedx/devstack`.

Reply:
I tried to test this in the rapid response repo which is also using `https://github.com/edx/devstack` repository. The test PR throws the same error in the [CI run](https://github.com/mitodl/rapid-response-xblock/actions/runs/8063963444/job/22026847471).

Reply:
Thanks for making this change.